client/blocks/reader-share/index.jsx

```diff
@@ -213,23 +214,32 @@ class ReaderShare extends React.Component {
 								position={ this.props.position }
 								className="reader-share__sites-popover"
 							/>
-						: <PopoverMenu
+						: <ReaderPopoverMenu
 								key="menu"
 								context={ this.refs && this.refs.shareButton }
 								isVisible={ this.state.showingMenu }
 								onClose={ this.closeExternalShareMenu }
 								position={ this.props.position }
 								className="popover reader-share__popover"
+								popoverTitle={ translate( 'Share on' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Share on:' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/share-component-to-use-reader-popover)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).